user_docs/en/userGuide.t2t

```diff
@@ -1280,6 +1280,9 @@ The following built-in Windows Console keyboard shortcuts may be useful when [re
 | Scroll down | control+downArrow | Scrolls the console window down, so later text can be read. |
 | Scroll to start | control+home | Scrolls the console window to the beginning of the buffer. |
 | Scroll to end | control+end | Scrolls the console window to the end of the buffer. |
+
+When using Windows Terminal version 1.18 and later, you may use NVDA's Move to previous page in review/Move to next page in review commands to scroll visible screens of output.

```

Comment:
Can you add a link to the relevant section of the user guide, and change the wording to improve readability?

```suggestion
When using Windows Terminal version 1.18 and later, the gestures ["Move to previous page in review" and "Move to next page in review" commands <add link to section of user guides>] can be used to scroll visible screens of output.

```

Reply:
this has not been addressed